tests/test_integration/test_frame_extraction.py

```diff
@@ -1,51 +1,490 @@
+import datetime
+import json
+import os
+import re
 from pathlib import Path
+from typing import Optional
 
 import pytest
+import typer
+from typer.testing import CliRunner
 
 from crabs.bboxes_labelling.extract_frames_to_label_w_sleap import (
     get_list_of_sleap_videos,
 )
 
 
-@pytest.fixture(autouse=True, scope="class")
-def input_video_dir():
-    return Path(__file__).parents[1] / "data" / "clips"
+@pytest.fixture()
+def input_data_dir() -> str:
+    return str(Path(__file__).parents[1] / "data" / "clips")
 
 
-class TestsFrameExtraction:
-    def test_extension_case_insensitive(self, input_video_dir):
-        """
-        Tests that the function that computes the list of SLEAP videos
-        is case-insensitive for the user-input extension.
+@pytest.fixture()
+def cli_inputs_dict(tmp_path: Path) -> dict:
```

Comment:
Is this a fixture taking `tmp_path` ... the fixture?!
TIL that this was possible.